### PR TITLE
Ftwd-miami- Brian Jaimes

### PR DESCRIPTION
**Pull request by BrianJaimes**
In-progress

### CODE REVIEW

File: starter-code/src/viking.js [hunk 1]
```diff
@@ -1,11 +1,39 @@
...
+function Viking(name,healthArg, strengthArg) {
+  Soldier.call(this, healthArg, strengthArg);
+  this.name = name;
+  console.log(this.name);
+  }
+  Viking.prototype = Object.create(Soldier.prototype);
+  Viking.prototype.constructor = Viking;
+  //var viking = new Viking (name, healthArg, strengthArg);
+  
+  var viking = new Viking (name, health, strength);
```
Comment (MarcosPalaciosIII) [inline starter-code/src/viking.js]: do not need to create this. The vikings and Saxons are created in the Jasmine tests.

File: starter-code/src/viking.js [hunk 2]
```diff
@@ -1,11 +1,39 @@
...
+  console.log(viking);
+  
+  
+  Viking.prototype.receiveDamage = function (damage) {
+    viking.health -= damage;
```
Comment (MarcosPalaciosIII) [inline starter-code/src/viking.js]: instead of viking.health should be this.health since your already calling the Viking.prototype

**Comment by MarcosPalaciosIII** [comment]
Remember not to delete the Jasmine files or comment out the tests. the jasmine test will actually guide you through the project as it tells you what it is looking for. Jasmine also sets the variables that you will be using for the tests so when using Jasmine you will not need to create a var for viking unless its something needed for the function.  If you need further explanation just come and ask me.  You were heading in the right track and I'm sure with more time you would have finished. Keep on truckin'

**Comment by B-riJ** [comment]
I get you, the Jasmine test parameters are beginning to look more familiar
now versus the binary code I was seeing it as before. Not having to create
variables unless they need to be for a function makes more sense now.
Thanks for the direction and feedback Marcos! Always appreciated. I'll keep
at it

On Fri, Apr 13, 2018 at 2:42 PM, MarcosPalaciosIII <notifications@github.com

**Pull request closed without merging**